spec/features/houses/update_house_spec.rb

```diff
@@ -0,0 +1,43 @@
+RSpec.describe "update house", type: :feature do
+  let!(:house1) do
+    FactoryBot.create(
+      :house,
+      number: "F 12",
+      address: "F alley",
+      route: "6th left",
+      status: "sold"
+    )
+  end
+
+  before do
+    visit root_url
+    click_on "House index"
+    click_on "Edit"
+  end
+
+  it "populates form with existing data" do
```

Comment:
ideally we should avoid extra `it`s in feature tests... in this case you could move all the expectations to L27